src/opers.c

```diff
@@ -146,135 +170,73 @@ void LoadFlags(
 **  the lower addressed half-word is the less significant
 **
 */
-#define HASH_FLAGS_SIZE (Int4)67108879L
+#define fnvp 1099511628211ULL
+#define fnvob 14695981039346656037ULL
```

Comment:
It's called the FNV hash. There will be some cost on a 32 bit system, but it's still only about 4 multiplications per byte hashed.  I'm not sure what happens if you are doing a 32 bit compile on a 64 bit capacble processor. The compiler mgiht be able to generate better code.

We can swap it out. Any usable non-cryptographic hash is fine. I like this one because it's simple.